apps/src/code-studio/components/header/SignInCalloutWrapper.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import cookies from 'js-cookie';
+import SignInCallout from './SignInCallout';
+
+const HideSignInCallout = 'hide_signin_callout';
```

Comment:
HideSignInCallout = false is a bit of a double-negative.  Would calling this value ShowSignInCallout be clearer?

Reply:
I thought about this for a while, and had the main flag called "showCallout" instead of "hideCallout", but then the cookie flag returns a 'false' if it hasn't been set yet, so we needed the cookies and session to return "true" otherwise, resulting in the "hideCallout". I initially had showCallout in the state, and hideCallout for the cookies and session, but as Maureen commented earlier, that led to a very logically confusing method.

Reply:
Ah, I see!  A brief comment about this might be a helpful hint to future readers of this code.

Reply:
Yes- good point. I'll try to add a comment explaining this (as brief and clear as possible)!